style.css

```diff
@@ -631,4 +649,55 @@ textarea {
   }
 }
 
+/* MQ para mobile js */
+@media only screen and (max-width: 375px) {
```

Comment:
- [ ] Kindly make sure that the mobile menu works as intended in the mobile view. Currently, you have this applies only to any width that is smaller than 375px since it's `max-width: 375px`. 
- Kindly make sure that this feature is working properly on widths `375px - 768px` which is for mobile view.